src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star,
+.stars--4 .stars__star:nth-child(-n +4),
+.stars--1 .stars__star:nth-child(-n +1),
+.stars--2 .stars__star:nth-child(-n +2),
+.stars--3 .stars__star:nth-child(-n +3) {
```

Comment:
Why not order from 1 to 5, looks confusing, and this should be avoided, especially in real projects